NEWS.md

```diff
@@ -2,7 +2,15 @@
 
 * Removed `CDISCFilteredDataset` class and functionality moved to `CDISCFilteredData`.
 * Changed constructor of `FilteredData` to not require `TealData` object. See `help(init_filtered_data)` for more details.
-* The filtered data is now stored in `FilteredData` not `FilteredDataset`. 
+* The filtered data is now stored in `FilteredData` not `FilteredDataset`.
+
+# Bug fixes
+
+* Fixed a bug when the filter panel overview would not refresh if the panel was hidden during a transition between active modules.
+
+# Miscellaneous
+
+* Enhanced the burger button so it is disabled when the filter panel is not used.
```

Comment:
burger button is not a concern of teal.slice

Reply:
transferred to teal